src/test/java/uk/gov/companieshouse/efs/api/events/service/fesloader/BatchDaoTest.java

```diff
@@ -49,7 +49,9 @@ void testBatchDaoObtainsNextBatchIdFromSequence() {
     @Test
     void testBatchDaoObtainsNextBatchNameIdFromSequence() {
         //given
-        when(this.template.queryForObject(anyString(), eq(Long.class), any())).thenReturn(BATCH_NAME_ID);
+        when(this.template.queryForObject(anyString(), eq(Long.class), any(Object[].class))).thenReturn(BATCH_NAME_ID);
+
+        //doReturn(BATCH_NAME_ID).when(this.template.queryForObject(anyString(), eq(Long.class), any()));
```

Comment:
Any reason this is commented out

Reply:
I was try to replace not working when() with doReturn() and forget to remove after that